astropy/modeling/core.py

```diff
@@ -69,9 +69,6 @@ class _ModelMeta(abc.ABCMeta):
     Currently just handles auto-generating the param_names list based on
     Parameter descriptors declared at the class-level of Model subclasses.
     """
-    @classmethod
-    def __prepare__(mcls, name, bases):
-        return OrderedDict()
 
```

Comment:
@nstarman  in #11017 modifies this method to add `**kwargs`. My understanding is that Python returns a dict by default if this method is not defined, so removing it here is fine. #11017 will need to be rebased on top of this before merging.

Reply:
Thanks for pointing that out.  I hand't seen that one.  Indeed, with Python 3.6 this `__prepare__` is no longer needed.